src/datapump/jobs/version_update.py

```diff
@@ -20,6 +21,7 @@ class RasterVersionUpdateJobStep(str, Enum):
     starting = "starting"
     creating_tile_set = "creating_tile_set"
     creating_tile_cache = "creating_tile_cache"
+    creating_aggregated_tile_set = "creating_aggregated_tile_set"
     creating_aux_assets = "creating_aux_assets"
     creating_cog_assets = "creating_cog_assets"
     mark_latest = "mark_latest"
```

Comment:
Please move the 'mark_latest' entry to between line 24 and 25.  That reflects what the state machine does, and what we require so that the aggregated tile set works correctly, and also the later aux_assets and cog_assets. 

We might eventually want to clean up so mark_latest is the last step (which we could do because we have the current version as a parameter).  Not sure why mark_latest wasn't always the last thing in the state machine.